### PR TITLE
Apply focus on Dynamo window after closing package manager window

### PR DESCRIPTION
**Pull request by yeexinc**
### Purpose

This PR is to apply focus on the Dynamo window when the "Online Package Search" window, "Installed Packages" window, or "Package Manager" window is closed.

The issue to address arises under the following sequence of events:
1. Have some other program opened and maximized (e.g. Google Chrome)
2. Open Dynamo and select Packages > Search for a Packages... / Manage Packages... / Package Manager to open a new window for managing packages
3. Minimize the package window
4. Reopen the package window
5. Close the package window. The Dynamo window becomes unfocused and is put behind the other program that has been maximized.

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions), and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@ellensi 

### FYIs

@Benglin

### CODE REVIEW

**Comment by ellensi** [comment]
Hi @yeexinc, thanks for going through this! Is there a JIRA task related to this pull request? If so, please add the reference link in the "Purpose" section of the PR description.

File: src/DynamoCoreWpf/Views/PackageManager/InstalledPackagesView.xaml.cs [hunk 1]
```diff
@@ -31,6 +31,9 @@ private void MoreButton_OnClick(object sender, RoutedEventArgs e)
             button.ContextMenu.DataContext = button.DataContext;
             button.ContextMenu.IsOpen = true;
         }
-
```
Comment (ellensi) [inline src/DynamoCoreWpf/Views/PackageManager/InstalledPackagesView.xaml.cs]: Let's make the habit to add a space between methods :)

File: src/DynamoCoreWpf/Views/PackageManager/PackageManagerSearchView.xaml.cs [hunk 2]
```diff
@@ -29,6 +29,7 @@ protected override void OnClosing(System.ComponentModel.CancelEventArgs e)
         {
             (this.DataContext as PackageManagerSearchViewModel).RequestShowFileDialog -= OnRequestShowFileDialog;
             base.OnClosing(e);
+            this.Owner.Focus();
```
Comment (ellensi) [inline src/DynamoCoreWpf/Views/PackageManager/PackageManagerSearchView.xaml.cs]: I believe adding this `Focus()` method will fix the window issue; thanks for that! Just one small thing: to make the behavior consistent, let's put the call to the `base` method at the end of this overriding method. (1) Unregister the registered events, (2) pass the focus to the owner window, and then (3) close this window.

**Comment by yeexinc** [comment]
@ellensi , thanks for looking into this! There was no JIRA task for this PR.

**Comment by ellensi** [comment]
Thanks for fixing this, @yeexinc!

**Pull request merged**